extension/background/youtubei.js

```diff
@@ -54,3 +54,83 @@ function parseTimestamp(ts) {
     const hours = parseInt(parts[2]) || 0
     return secs + (60 * mins) + (60 * 60 * hours)
 }
+
+// time-comments related code below
+
+const MAX_COMMENT_PAGES = 5
+const MAX_COMMENTS = 100
+
+const INNERTUBE_API_KEY = "AIzaSyAO_FJ2SlqU8Q4STEHLGCilw_Y9_11qcW8"
+const INNERTUBE_CLIENT_NAME = "WEB"
+
+export async function fetchComments(videoId) {
+    const videoResponse = await fetchVideo(videoId)
+
+    let token = commentsContinuationToken(videoResponse)
+    if (!token) {
+        return []
+    }
+    const comments = []
+    let prevToken
+    let pageCount = 0
+    while (prevToken !== token && pageCount < MAX_COMMENT_PAGES && comments.length < MAX_COMMENTS) {
```

Comment:
There are about 5 HTTP requests. The timestamps we care about probably are in the very first comment. It seems like we could make just one to get the first page.

Reply:
There is no need in those constants and the loop. Just make only one request.